src/lib/pickles/fix_domains.mli

```diff
@@ -1,5 +1,6 @@
 val domains :
-     (module Snarky_backendless.Snark_intf.Run with type field = 'field)
+     ?min_log2:int
```

Comment:
Could we maybe document how this new argument should be used ? (and its default value to 0).